src/test/EigenPod.t.sol

```diff
@@ -853,40 +854,6 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
         newPod.verifyBalanceUpdates(uint64(block.timestamp - 1), validatorIndices, stateRootProofStruct, proofs, validatorFieldsArray);
     }
 
-    //test that when withdrawal credentials are verified more than once, it reverts
-    function testDeployNewEigenPodWithActiveValidator() public {
```

Comment:
Looking through the diffs, can't see where this is added back - will you move it to integration tests?

Reply:
Added here since it was more of a unit test: 

https://github.com/Layr-Labs/eigenlayer-contracts/blob/ec5cd3037ecb73e69f044dfe5b0477fe985c9f3e/src/test/unit/EigenPodUnit.t.sol#L397-L416